js/plates-view/render-cross-section.ts

```diff
@@ -478,19 +478,26 @@ class CrossSectionRenderer {
       if (metamorphic > 0) {
         // "Vertical" metamorphism.
         if (field.subduction && !field.canSubduct) {
-          // points that are subducting, but shouldn't, will skip the first metamorphic color (transparent).
-          // They're deeper, so the visualization looks better when we start from the second shade to match
-          // neighboring points better.
-          // Divide vertical p1-p4 line into 3 sections (p1-p1a, p1a-p1b, p1b-p4).
-          const p1a = (new THREE.Vector2()).lerpVectors(p1, p4, METAMORPHISM_OROGENY_COLOR_STEP_0);
-          const p1b = (new THREE.Vector2()).lerpVectors(p1, p4, METAMORPHISM_OROGENY_COLOR_STEP_1);
-          // Divide vertical p2-p3 line into 3 sections (p2-p2a, p2a-p2b, p2b-p3).
-          const p2a = (new THREE.Vector2()).lerpVectors(p2, p3, METAMORPHISM_OROGENY_COLOR_STEP_0);
-          const p2b = (new THREE.Vector2()).lerpVectors(p2, p3, METAMORPHISM_OROGENY_COLOR_STEP_1);
-
-          this.fillMetamorphicOverlay("Low Grade Metamorphic Rock (Continental Collision)", field, p1, p2, p2a, p1a);
-          this.fillMetamorphicOverlay("Medium Grade Metamorphic Rock (Continental Collision)", field, p1a, p2a, p2b, p1b);
-          this.fillMetamorphicOverlay("High Grade Metamorphic Rock (Continental Collision)", field, p1b, p2b, p3, p4);
+          const kMetamorphismHighGradeOnlyThreshold = 0.015;
```

Comment:
I take it that you don't think it's worth elevating this to a `config` parameter?

Reply:
Yes, as this value is pretty much about filtering out the first field that is just subducting. So, there's not that much to configure and/or play with I guess.